h/search/persistence.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+"""Code for persisting (indexing) annotations to Elasticsearch."""
+
+from __future__ import unicode_literals
+
+import elasticsearch_dsl
+
+
+class Annotation(elasticsearch_dsl.DocType):
+    """An elasticsearch_dsl annotation document."""
+
+    class Meta:
+        index = "hypothesis"
+
+    authority = elasticsearch_dsl.Keyword(index=False)
+
+    @staticmethod
+    def create(models_annotation):
```

Comment:
The naming of this param `models_annotation` gave me pause: it sounds plural when I presume we'd be dealing with a single annotation model? I grok that it's derived from `models.Annotation` and perhaps it's a standard param-naming pattern, in which case, ignore me.